src/app/cmd/clear_image.cpp

```diff
@@ -19,15 +19,16 @@ namespace cmd {
 
 using namespace doc;
 
-ClearImage::ClearImage(Image* image, color_t color)
-  : WithImage(image)
-  , m_color(color)
+ClearImage::ClearImage(Cel* cel, color_t color)
```

Comment:
I'm not sure if there is another solution (I will try to suggest something) but I don't think we should change the image parameter by a cel...it just doesn't feel right...ClearImage should clear an Image IMO.

Reply:
Now thinking about it - maybe I could just save the frame and restore it in ClearCel since ClearCel calls ClearImage anyways?

Reply:
That sounds better, but I'm still wondering if there is an easier fix. Anyway David said that he will review this.